doc/source/whatsnew/v0.21.0.txt

```diff
@@ -260,7 +260,7 @@ Conversion
 
 - Bug in assignment against datetime-like data with ``int`` may incorrectly converte to datetime-like (:issue:`14145`)
 - Bug in assignment against ``int64`` data with ``np.ndarray`` with ``float64`` dtype may keep ``int64`` dtype (:issue:`14001`)
-
+- Bug in :func:`to_datetime` incorrectly localizing dates in `Series` when `utc=True` (:issue:`6415`)
```

Comment:
I feel like this an API breaking change.  Enhancement implies it adds functionality that's beneficial and isn't patching existing behavior *per se*.